### DISCUSSION TITLE
Actividad 1 - GRUPO 9

### DISCUSSION THREAD

**Discussion by EmilianoColavita** [frlp-utn-ingsoft/catedra#98] [Show and tell]
https://github.com/EmilianoColavita/vetsoft

**Comment by RodrigoJacznik** [comment]
Buenas!
La actividad está bien resuelta, queda aprobada.
Muy bueno el trabajo que hicieron agregando los fixtures y como normalizaron los mensajes de commits 👏 👏 

Les dejamos algunos comentarios para que tengan en cuenta en la próxima:

- Hay algunos tests que quedaron rotos. Como la app funciona y además esta actividad no se trataba de tests, no hay problema, pero a partir de la próxima actividad no puede quedar ningún test fallando.

Buen trabajo!
Saludos
